.coderabbit.yaml

```diff
@@ -0,0 +1,81 @@
+# Docs: https://docs.coderabbit.ai/configure-coderabbit
+# Schema: https://coderabbit.ai/integrations/schema.v2.json
+
+language: en
+
+tone_instructions: |
```

Comment:
Is there a way we can limit the number of comments it makes per-PR? My ideal here is that we can limit the amount of noise it creates. 

In thinking about this + my other comment below... I think we should try to engage with either CR support or their community to understand what is possible on both of these fronts. @gberenice can you look into that?

Reply:
Considering we already use "chill" profile + disable some requests/comments, I'd avoid this right now.
* There is no dedicated setting to achieve that.
* `tone_instructions` and/or `path_instructions` could do the trick, but again, it's _likely_ (like with the links to the docs).
* I don't have a clear understanding how it's going to sort the comments. I'm concerned we can miss something valuable.

Let's give it a chance and see how it goes. How does that sound?

Reply:
@gberenice sounds good to me 👍